site_scons/components/__init__.py

```diff
@@ -312,7 +312,10 @@ def define_components(reqs):
     retriever = GitRepoRetriever("https://github.com/spdk/spdk.git", True)
     reqs.define('spdk',
                 retriever=retriever,
-                commands=['./configure --prefix="$SPDK_PREFIX"'                \
+                commands=['git config user.email "daos@daos.groups.io"',
```

Comment:
does this change local settings?  Why do we need to add git config here anyway?

We already have a built-in mechanism for applying a patch if it's helpful

In build.config
[patch_versions]
spdk=https://path/to/patch